kcidb/templates/stable_rc_test.j2

```diff
@@ -23,15 +23,13 @@
                 selectattr('build.architecture', 'ne', None) | list %}
                 {% if boot_tests_info %}
                     {% for architecture, tests in boot_tests_info|groupby("build.architecture") %}
-                        {{- "      " + architecture }}:({{ tests|map(attribute="build.config_name") | unique | join("") }})
-                        {% for test in tests %}
-                            {{- "      -" + test.environment_misc.platform }}
-                        {% endfor %}
+                        {{- "\n      " + architecture }}:({{ tests|map(attribute="build.config_name") | unique | join("") }})
+                        {{- "\n      -" + tests|map(attribute="environment_misc.platform") | unique | join("\n      -") }}
```

Comment:
Do you wanna drop `None` configs platforms here, perhaps?

Reply:
I updated to drop `None` build configs. `None` platforms would be handled at https://github.com/kernelci/kcidb/blob/main/kcidb/templates/stable_rc_test.j2#L22.